pkg/vault/test/foundry/ProtocolFeeController.t.sol

```diff
@@ -486,6 +486,10 @@ contract ProtocolFeeControllerTest is BaseVaultTest {
         feeController.withdrawPoolCreatorFees(pool, alice);
     }
 
+    function testPermissionlessWithdrawalByNonPoolCreator() public {
+        feeController.withdrawPoolCreatorFees(pool);
```

Comment:
Maybe check that they were effectively collected?

Reply:
I was afraid somebody might say that :) Both call a common routine, so it's pretty clear by inspection that it will work, but sure, I'll add the test. (After all, the code could change diverge later, and if does something should break.)